src/TLuaInterpreter.cpp

```diff
@@ -10879,6 +10884,21 @@ bool TLuaInterpreter::compile(const QString& code, QString& errorMsg, const QStr
     }
 }
 
+// returns true if the given Lua code is valid, false otherwise
+bool TLuaInterpreter::validLuaCode(const QString &code)
+{
+    lua_State* L = pGlobalLua;
+    if (!L) {
+        qWarning() << "LUA CRITICAL ERROR: pGlobalLua Lua execution unit found.";
```

Comment:
Um, should there not be a "not" in that error message? 😜

Reply:
Good catch! Thanks, fixed.